observation_portal/observations/realtime.py

```diff
@@ -120,8 +121,12 @@ def get_realtime_availability(user: User, telescope_filter: str = ''):
     for resource in telescopes_availability.keys():
         telescope, enclosure, site = resource.split('.')
         intervals = filtered_dark_intervalset_for_telescope(start, end, site, enclosure, telescope)
-        # Now also filter out running obs, future high priority (TC, RR, Direct) obs, and obs overlapping in time by the user
         intervals_to_block = []
+        # Filter out according to our custom per telescope rules
+        intervals_to_block.append(import_string(settings.OVERRIDES['realtime_intervals_to_block_for_telescope'])(
```

Comment:
This method is called to get all the possible realtime windows a user could book